mitosheet/src/mito/components/toolbar/GraphTabs/GraphTypeConfigurations.tsx

```diff
@@ -27,22 +27,22 @@ export const GraphTypeConfigurations = (
                         if (newNumberBins === undefined || +newNumberBins < 0) return;
                         props.updateGraphParam({graph_creation: {nbins: +(newNumberBins ?? '')}})
                     }}
-                    width='small'
+                    style={{ width: '70px' }}
                 />
             </div>);
         }
         if (GRAPHS_THAT_HAVE_HISTFUNC.includes(props.graphParams.graph_creation?.graph_type)) { 
             configurationInputs.push(<div className="mito-graph-configuration-option">
-                <LabelAndTooltip tooltip='The metric displayed for each bin of data' textBody>
+                <LabelAndTooltip style={{ fontWeight: 'bold' }} tooltip='The metric displayed for each bin of data' textBody>
                         Aggregation Type
                 </LabelAndTooltip>
                 <Select
                     value={props.graphParams.graph_creation?.histfunc || 'count'}
                     onChange={(newHistfunc: string) => {
                         props.updateGraphParam({graph_creation: {histfunc: newHistfunc}})
                     }}
-                    width='medium'
-                    dropdownWidth='medium'
+                    style={{ width: '70px' }}
```

Comment:
Can we make this a class, just so we can easily keep the width consistent across the different graph configs

Reply:
Ok so I tried this out and the width of the other config option (points) was better as-is because it actually needs to be wider than these. I vote we keep these hard-coded here.

Reply:
Sounds good!